source/sass/views/home.scss

```diff
@@ -114,3 +114,64 @@
     }
   }
 }
+
+/**
+* Partner section
+*/
+.partner-section {
+  background-size: cover;
+  background-position: center center;
+  position: relative;
+  z-index: 10;
+  padding-top: 80px;
+  padding-bottom: 80px;
+}
+
+.partner-section::before {
+  content: '';
+  position: absolute;
+  z-index: 0;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.6);
+}
+
+.partner-section-text {
+  font-family: $font-family-sans-serif;
```

Comment:
Let's make use of the two mixins `text-base-style` and `set-text-size` to set font style here.